module/x/gravity/types/msgs.go

```diff
@@ -320,6 +320,8 @@ type EthereumClaim interface {
 	// validators claims agree. Therefore it's extremely important that this include all elements of the claim
 	// with the exception of the orchestrator who sent it in, which will be used as a different part of the index
 	ClaimHash() ([]byte, error)
+	// Sets the orchestrator value on the claim
+	SetOrchestrator(sdk.AccAddress)
```

Comment:
Go generics leave much to be desired, so I added this function to the EthereumClaim interface so that I could set the Orchestrator on any EthereumClaims I generate in the tests.

Reply:
is there any way we can make it panic when used outside of tests?

Reply:
We discussed separately that this is fine to leave in due to the lack of conditional compilation tools